### ISSUE TITLE
Audio exported to .m4a files don't accept cover images in Apple Music

### ISSUE THREAD

**Issue by bouscram** [mifi/lossless-cut#1468]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md)
- [X] Search for your problem under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)
- [X] If you have an issue only with one particular file, [use this form instead](https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&template=file-issue.yml)

### Operating System

MacOS 13

### Steps to reproduce

From a .mp4 file created by Handbrake from a DVD, try to extract the audio track as .m4a files, with the "Preserve all MP4/MOV metadata? " flag to "No", as recommended.

I use the chapter track in the file to separate the songs, and keep only the audio track selected, as shown below.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/1451624/218306077-708e117d-10da-420a-aeba-5ef0024c1453.png">

### Expected behavior

The extracted files should behave as any other .m4a audio file when imported to Apple Music or iTunes

### Actual behavior

After the file is imported to Apple Music, it is impossible to add a cover picture to the songs.

I did several tests:

1) with the "Metadata" flag to "Off", it is completely impossible to paste an image in the relevant dialogue. Nothing shows
2) with the "Metadata" flag to "On", it is possible to paste an image, but it is not kept
3) I used Subler to remove the "Menu" track from a file created for 2), and re-save it (I guess recreating the MP4 container), and the produced file worked in Music.

I guess there is something in the format of th MP4 file produced by LosslessCut which causes problem to Music.

**Comment by bouscram** [comment]
A small update: step 3 above works only if you drag the file to a "new" Subler document. If you open the file with Subler, remove the extra track and save, it is like 1) or 2) depending on the file

**Comment by bouscram** [comment]
2nd update : the issue has nothing to do with the extra menu track... It's only something with the file format, apparently...

**Comment by mifi** [comment]
You are running the latest version, right?

Just to be sure, you use the normal Export button right? (not "extract this track as a file" in the track panel)

I'm not able to reproduce the problem unfortunately. I tried:

- Open mp4 file in losslesscut with aac audio
- Disable video track so only audio track remains
- Export
- Open Apple Music
- Drag-drop exported .mp4 file into albums
- Get info -> Artwork - Add Artwork
- Quit and reopen Apple Music
- See the thumbnail is still there

are you using MP4 as output format? what if you try `ipod` instead? related: #331

also which version of macos are you running? ventura 13.1 here

**Comment by bouscram** [comment]
That's strange... I'm not at home right now, but at least I can tell that I use the latest version, on Ventura 13.1 (but I also checked on an older Mac which is still running iTunes and the behaviour is the same). I will provide you with a "problematic file" this evening if you want.
I used the "iPod" export format (=> .m4a) (and I didn't try with the default .mp4 format). Maybe you can try that to see if the problem is there...

**Comment by mifi** [comment]
haha i also didn't notice it was you who posted that one! cannot reproduce with the ipod export format either, but if you can provide me with the file that would help

**Comment by bouscram** [comment]
I did some additional tests today, it it appears that the culprit is my original file. It's from a DVD I converted to mp4 10 years ago, and there is something strange with it. 
I tried with another one I did this week, and there is no problem (as it was for you).

I will close the issue, but I will also attach 2 files extracted from the problematic one in a zip, if you want to further test: the 1st one (1--Title-.m4a) has been created with the "preserve all metadata" option on and it's possible to import it to Music, but impossible to add artwork. The 2nd (1-Psycho Killer.m4a) has been created with  "preserve all metadata" option off, at it's not even possible to import it to Music...

[Archive.zip](https://github.com/mifi/lossless-cut/files/10774957/Archive.zip)
